projectDocs/issues/triage.md

```diff
@@ -143,13 +143,22 @@ An issue is triaged if it is ready to be worked on.
 New features and enhancements should be [well defined](#new-features-1) before applying the `triaged` label.
 Once a bug has clear steps to reproduce and is well documented, the `triaged` label can be applied.
 A `triaged` issue should also have a [priority label](#priority).
-Generally these labels are provided by NV Access only, particularly for issues that are controversial, or priority is unclear.
 
-For controversial changes, a product decision from NV Access may be required before applying the `triaged` label.
-This can be indicated with adding the label `blocked/needs-product-decision`.
+Community members should avoid adding a `triaged` label to feature requests or other decisions potentially involving significant or controversial changes to NVDA features or functionality, which may require community input or approval from NV Access.
+The label should also be avoided for other issues that are controversial, or where the priority is unclear, such as bug fixes with an unclear solution.
+For changes where a product decision from NV Access is required before applying the `triaged` label, the label `blocked/needs-product-decision` should be used.
+Community members should apply the `triaged` where an issue is:
+
+* A well formed bug report following the issue template
+* Which can easily be understood
+* Isn't missing any important debug information
+* Can clearly be reproduced
+* Can clearly be prioritised
+* Doesn't require a controversial solution
```

Comment:
Removing it would create a more vague set of expectations. Could you suggest what's missing or not objective? We tend to follow this process more-or-less when triaging issues as NV Access. In our experience its quite clear and objective if something can be reproduced, has attached debug logs, and is prioritised in our label system.

Reply:
see https://github.com/nvaccess/nvda/pull/17315#discussion_r1839419962